go.mod

```diff
@@ -2,11 +2,6 @@ module github.com/carousell/Orion
 
 go 1.17
 
-replace (
-	git.apache.org/thrift.git => github.com/apache/thrift v0.12.0
-	github.com/openzipkin-contrib/zipkin-go-opentracing => github.com/openzipkin-contrib/zipkin-go-opentracing v0.3.5
```

Comment:
`github.com/openzipkin-contrib/zipkin-go-opentracing@v0.3.5` is explicitly listed in this go.mod. no `replace` is needed to override the version